### ISSUE TITLE
Problem loading tests with advancedExecutables env since 4.0.2

### ISSUE THREAD

**Issue by clipitar** [matepek/vscode-catch2-test-adapter#301]
**Checklist**

- [x] The issue is about this extension and NOT about a fork.
- [x] Check the [known issues](https://github.com/matepek/vscode-catch2-test-adapter/blob/master/documents/support.md#known-issues) list.
- [x] The latest version of the extension was used.
- [x] It is not related to remote-vscode or I checked the following [issue](https://github.com/matepek/vscode-catch2-test-adapter/issues/201)

**Describe the bug**

Since TestMate has updated to 4.0.2 (same behaviour for 4.0.2-4.0.5), my unit tests are no longer loaded in the Test Explorer panel.

My test executables are configured as such through the testMate.cpp.test.advancedExecutables option:
```
"testMate.cpp.test.advancedExecutables": [
		{
			"pattern": "bin/Debug/**/*validate_*",
			"env": {
				"LD_LIBRARY_PATH": "${os_env:HOME}/Vault/ubuntu18_gcc9/poco/poco_1.9.3/lib"
			},
			"sourceFileMap": {
				"/home/user/git/std_c": "${os_env:HOME}/git/std_c"
			}
		}
	],
```

Which has been working fine until updating to 4.0.2.

Supplied environment variable LD_LIBRARY_PATH no longer seems to be loaded in ExecutableConfig.resolveVariable (see attached logs), hence test executables cannot load because they cannot find their dependency.

I have tried specifying an absolute path for my environment variable, in case that's what was causing trouble (as it was the case in not so distant versions of vscode), but I get the same behaviour.

**To Reproduce**

I guess the best way of reproducing this would be to have a test executable depending on a shared library that's found outside of the workspace, and use 
```
"env": {
	"LD_LIBRARY_PATH": "/path/to/lib"
},
```
to link the executable to its library.  
I'm not saying this is a good design, but that's another topic...!

**Desktop**

- Extension Version: 4.0.2
- VS Code Version: 1.62.3
- Catch2 / Google Test / DOCTest Version: GoogleTest 1.8
- OS Type and Version: Linux x64 5.4.0-90-generic, Ubuntu 20.04
- Using remote-ssh/docker/wsl?: none of these, working locally on Ubuntu vm.

**Regression bug?**

This seems to be regression, since it worked well in 4.0.1

- Last extension version in which the feature were working: 4.0.1

**Log** (optional but recommended)
[log-4.0.5-not-working.txt](https://github.com/matepek/vscode-catch2-test-adapter/files/7605080/log-4.0.5-not-working.txt)
[log-4.0.5-not-working-absolute-path.txt](https://github.com/matepek/vscode-catch2-test-adapter/files/7605081/log-4.0.5-not-working-absolute-path.txt)
[log-4.0.1-working.txt](https://github.com/matepek/vscode-catch2-test-adapter/files/7605082/log-4.0.1-working.txt)

**Comment by matepek** [comment]
will be fixed in next release (v4.0.6)

**Comment by clipitar** [comment]
I confirm that it's fixed.  Thanks a lot!!
